openfisca_core/periods/_parsers.py

```diff
@@ -34,7 +34,7 @@ def _parse_period(value: str) -> Optional[Period]:
         return None
 
     # Check for a non-empty string.
-    if not (value and isinstance(value, str)):
+    if len(value) == 0:
```

Comment:
Are we completely sure that value will be a string ?

Reply:
No, it just tests for the declared types in the method signature. I willl make it broader, just in case.

Reply:
Actually, yes. This is an internal function, so we're sure these are `str`.